router/server.go

```diff
@@ -67,9 +68,19 @@ func main() {
 	tcpIP := flag.String("tcpip", "", "tcp router listen ip")
 	tcpRangeStart := flag.Int("tcp-range-start", 3000, "tcp port range start")
 	tcpRangeEnd := flag.Int("tcp-range-end", 3500, "tcp port range end")
+	certFile := flag.String("tlscert", "", "TLS (SSL) cert file in pem format")
+	keyFile := flag.String("tlskey", "", "TLS (SSL) key file in pem format")
```

Comment:
This is good for anyone running the router standalone, but won't work inside of Flynn as there's no good way to inject config files into a container currently. We should support environment variables as well (which is what we're currently using for secrets in lieu of a more formal/secure system).